### PR TITLE
Edits

### PR DESCRIPTION
**Pull request by Phenomenalhub**

### CODE REVIEW

File: app/src/main/java/com/example/studentarena/MainActivity.java [hunk 1]
```diff
@@ -18,14 +19,18 @@
 import com.google.android.material.bottomnavigation.BottomNavigationView;
 
 public class MainActivity extends AppCompatActivity {
-    //Toolbar toolbar;
+
+    // https://maps.googleapis.com/maps/api/geocode/json?address=1600+Amphitheatre+Parkway,+Mountain+View,+CA&key=YOUR_API_KEY
+
+    //String Address = "", state = "", city = "";
+
+    //String getLocationRequest = "https://maps.googleapis.com/maps/api/geocode/json?address=" + Address + ",+" + state;
```
Comment (Phenomenalhub) [inline app/src/main/java/com/example/studentarena/MainActivity.java:27]: Trying to work on location when I made the commits.

File: app/src/main/res/layout/fragment_feed.xml [hunk 2]
```diff
@@ -62,15 +63,15 @@
 
     <androidx.recyclerview.widget.RecyclerView
         android:id="@+id/rvPosts"
-        android:layout_width="402dp"
-        android:layout_height="668dp"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
```
Comment (mhammerly) [inline app/src/main/res/layout/fragment_feed.xml:67]: just double-checking - is this intentional? do you know why these are supposed to be 0dp?

File: app/src/main/res/layout/activity_detail.xml [hunk 3]
```diff
@@ -45,31 +45,31 @@
...
+            android:text="Title"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toBottomOf="@+id/ivImage" />
 
         <TextView
             android:id="@+id/tvPrice"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:text="TextView"
+            android:text="$"
```
Comment (mhammerly) [inline app/src/main/res/layout/activity_detail.xml:64]: per sushant's comment on the other PR, should you move these values to `strings.xml`?

File: app/src/main/res/layout/fragment_feed.xml (hunk 2 above)
Comment (Phenomenalhub) [reply app/src/main/res/layout/fragment_feed.xml:67]: Yes it was. A constraint not to have any space in the height and width.

Comment (Phenomenalhub) [reply app/src/main/res/layout/fragment_feed.xml:67]: Yes it was. A constraint not to have any space in the height and width.

File: app/src/main/res/layout/fragment_feed.xml [hunk 4]
```diff
@@ -62,15 +63,15 @@
...
         android:layout_centerInParent="true"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintHorizontal_bias="0.0"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/postToolbar"
-        app:layout_constraintVertical_bias="0.8" />
+        app:layout_constraintVertical_bias="0" />
```
[the remark this answers is not in the document]
Comment (Phenomenalhub) [reply app/src/main/res/layout/fragment_feed.xml:74]: Both constraintVertical_bias="0" and constraintHorizontal_bias="0.0" are not needed it happens when I drag it so i removed them.

File: app/src/main/res/layout/activity_detail.xml [hunk 5]
```diff
@@ -45,31 +45,31 @@
...
+            app:layout_constraintTop_toBottomOf="@+id/tvContactInfo"
             tools:text="Welcome to student arena" />
 
         <TextView
             android:id="@+id/tvTitle"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_marginTop="8dp"
-            android:text="TextView"
+            android:text="Title"
```
[the remark this answers is not in the document]
Comment (Phenomenalhub) [reply app/src/main/res/layout/activity_detail.xml:56]: Will work on them then commit and merge.

**Pull request merged**